ingest/src/main/java/com/connexta/ingest/client/StoreClient.java

```diff
@@ -71,8 +70,8 @@ public String getFilename() {
     final URI location;
     try {
       location = restTemplate.postForLocation(storeEndpoint, request);
-    } catch (final RestClientException e) {
-      throw new StoreException("Unable to POST to store endpoint", e);
+    } catch (Exception e) {
+      throw new StoreException(e);
```

Comment:
❓ Why remove the additional info that was added which brings a bit more detail about the condition in which the error occurred?

Reply:
I can provide a a blanket answer to all of your comments in this review:

I suggested to the team in the mob review that we keep things simple and easier to maintain. We were't using a lot of this functionality yet, so I opted to remove it for now and add it back in once we are using these exceptions as intended. We will create tickets to improve the message and details for different scenarios. This PR just adds the groundwork.

Reply:
I am just concerned that this is adding technical debt on some trivial things that should normally be done as you write code. Things like piggy backing a cause exception is part of the guidelines and I am not sure we should skip this as we will definitely forget about that.